src/discord/commands/book.ts

```diff
@@ -48,14 +49,14 @@ function getDateOptions(selectedDate?: string) {
 async function parseCommandOptions(interaction: CommandInteraction): Promise<string[] | undefined> {
     //Parses the book command option parameters to return the corresponding Room's section ID
     const roomName = interaction.options.getString('room-name');
-    const roomJson = await roomModel.findOne({ name: roomName !== null ? roomName : undefined });
+    const roomJson = await RoomModel.findOne({ name: roomName !== null ? roomName : undefined });
 
     if (roomJson === null) {
```

Comment:
```suggestion
    if (!roomJson) {
```

Use an undefined check as well so we don't need a non-null assertion later on.